libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.html

```diff
@@ -81,41 +80,41 @@
           </sky-toolbar-view-actions>
         </sky-toolbar-section>
       </sky-toolbar>
-      <div
-        *ngIf="!isSearching && displayedItems.length === 0"
-        class="sky-font-deemphasized sky-lookup-show-more-no-results"
-      >
-        {{ 'skyux_lookup_show_more_no_results' | skyLibResources }}
-      </div>
-      <sky-repeater class="sky-lookup-show-more-repeater" expandMode="none">
-        <sky-repeater-item
-          *ngFor="let item of displayedItems"
-          [isSelected]="
-            item[context.idProperty] | skyLookupItemSelected: selectedIdMap
-          "
-          [selectable]="context.selectMode === 'multiple'"
-          (click)="itemClick(item)"
-          (keyup.enter)="itemClick(item)"
-          (isSelectedChange)="onItemSelect($event, item)"
-        >
-          <sky-repeater-item-content>
-            <ng-container
-              *ngTemplateOutlet="
-                repeaterItemTemplate ?? defaultItemTemplate;
-                context: { item: item }
-              "
-            >
-            </ng-container>
-          </sky-repeater-item-content>
-        </sky-repeater-item>
-      </sky-repeater>
-      <sky-infinite-scroll
-        [enabled]="hasMoreItems"
-        [loading]="isLoadingMore"
-        (scrollEnd)="infiniteScrollEnd()"
-      >
-      </sky-infinite-scroll>
     </div>
+    <div
```

Comment:
from what I can tell this code is the same, it's just the indentation that's different?? I might be missing the changed line.

Reply:
Previously, the `<div>` for "show-no-more-results" and the accompanying `<sky-repeater>` and `<sky-infinite-scroll>` elements were children of the `<div class="sky-lookup-show-more-toolbar-wrapper">` element.  Now they are siblings of that element and all children of the `<sky-modal-content>` element.  So it's more than just indentation - it's the parent/child relationship of the elements.